src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioErrorReportingService.cs

```diff
@@ -31,43 +30,72 @@ internal partial class VisualStudioErrorReportingService : IErrorReportingServic
             _listener = listener;
         }
 
-        public void ShowErrorInfo(string message, params ErrorReportingUI[] items)
+        public void ShowErrorInfoInActiveView(string message, params ErrorReportingUI[] items)
```

Comment:
I think for things like code fix failure, using active view is better since the failure is actually related to that active view.